util/resolution_analysis.py

```diff
@@ -374,25 +377,31 @@ def miller_array_from_mtz(unmerged_mtz, anomalous=False, labels=None):
   reflections_per_bin = 10
     .type = int
     .help = "Minimum number of reflections per bin."
+    .short_caption = "Minimum number of reflections per bin"
   binning_method = *counting_sorted volume
     .type = choice
     .help = "Use equal-volume bins or bins with approximately equal numbers of reflections per bin."
     .short_caption = "Equal-volume or equal #ref binning."
     .expert_level = 1
   anomalous = False
     .type = bool
-    .short_caption = "Keep anomalous pairs separate in merging statistics"
+    .help = "Keep anomalous pairs separate in merging statistics"
+    .short_caption = "Anomalous"
     .expert_level = 1
   labels = None
     .type = strings
+    .short_caption = "Labels"
   space_group = None
     .type = space_group
     .expert_level = 1
+    .short_caption = "Space group"
   reference = None
     .type = path
+    .short_caption = "Reference"
   emax = 4
     .type = float(value_min = 0)
-    .help = "Reject reflecitons with normalised intensities E^2 > emax^2"
+    .help = "Reject reflections with normalised intensities E^2 > emax^2"
+    .short_caption = "Maximum normalised intensity E"
```

Comment:
"E" probably doesn't add anything here...
```suggestion
    .short_caption = "Maximum normalised intensity"
```